### DISCUSSION TITLE
Ethernet version?

### DISCUSSION THREAD

**Discussion by JackBaltzer** [UtilitechAS/amsreader-firmware#450] [Ideas]
Greetings,

I have been using this software on a ESP32 for some time, but have issues with the wifi signal reaching into the powermeter box, which means the dataflow breaks occasionally.
Because of that I have bought a OLIMEX ESP32 POE board: [https://www.olimex.com/Products/IoT/ESP32/ESP32-POE/open-source-hardware](https://www.olimex.com/Products/IoT/ESP32/ESP32-POE/open-source-hardware) hoping I could install the software on it and it would work... I could not.

Would it be possible to build an Ethernet based version of this soffware? 

I am a absolute newbie when it comes to Arduino/PlatformIO and I have no idea where to start and what to change. I have been using the esptool flasher with the prebuild releases.

**Comment by gskjold** [comment]
You are not alone in wanting this feature :) Right now the firmware does not support ethernet, but this will be addressed at some point. Follow issue #365 to receive updates when I start working on this. I will provide beta builds for testing through that issue.
